requirements.txt

```diff
@@ -1,2 +1,3 @@
 urllib3 >= 1.25
 requests
+fsspec
```

Comment:
Is it fine to take dependency on fsspec? Is it a commonly used/supported package?